core/tests/protractor_utils/general.js

```diff
@@ -33,6 +33,14 @@ var checkForConsoleErrors = function(errorsToIgnore) {
   var irrelevantErrors = errorsToIgnore.concat(CONSOLE_ERRORS_TO_IGNORE);
   browser.manage().logs().get('browser').then(function(browserLogs) {
     var fatalErrors = [];
+    // The mobile tests run on the latest version of Chrome.
+    // The newer versions report 'Slow Network' as a console error.
+    // This causes the tests to fail, therefore, we remove such logs.
+    if (browser.isMobile) {
+      browserLogs = browserLogs.filter(function(browserLog) {
+        return !(browserLog.message.includes(' Slow network is detected.'));
```

Comment:
We compiled the logs from Browserstack (can be found out [here](https://gist.github.com/apb7/a00ef6c4ae81bf468d886b67d68ee57b)) (Thanks @lilithxxx!). I observed a common pattern ` Slow network detected.` and therefore filtered out such logs in case of mobile tests.